plugins/scaffolder/src/components/ListTasksPage/ListTasksPage.tsx

```diff
@@ -51,23 +51,29 @@ export interface MyTaskPageProps {
 const ListTaskPageContent = (props: MyTaskPageProps) => {
   const { initiallySelectedFilter = 'owned' } = props;
   const { t } = useTranslationRef(scaffolderTranslationRef);
+  const [limit, setLimit] = useState(5);
+  const [page, setPage] = useState(0);
```

Comment:
@drodil I wonder if the initial page "0" is better than "1"?

Usually on UI user see the numeration of pages started with 1, not 0.
Won't it lead in the future to do some kind of conversion (N-1) to get proper offset?

Reply:
Actually no. The MUI table uses zero-based index for the page, see here: https://mui.com/api/table-pagination/